tests/test_utils.py

```diff
@@ -58,3 +60,46 @@ def test_upload_dir_to_s3(tmp_path):
                 call(tmp_path / 'subdir1/subdir3/bar.txt', 'myBucket', 'myPrefix/subdir1/subdir3/bar.txt'),
             ]
         )
+
+
+@mock_aws
+def test_list_files_in_s3(s3_bucket):
+    prefix = 'geotiffs'
+    geotiffs = [
+        f'{prefix}/my-file1.tif',
+        f'{prefix}/my-file2.tif',
+        f'{prefix}/my-file3.tif',
+    ]
+
+    for tif in geotiffs:
+        ut.S3_CLIENT.put_object(Bucket=s3_bucket, Key=tif)
+
+    files = ut.list_files_in_s3(s3_bucket, prefix)
+
+    assert len(files) == len(geotiffs)
+    assert files[0]['Key'].endswith('.tif')
+
+
+@mock_aws
+def test_download_file_from_s3(tmp_path, s3_bucket):
+    download_path = 'geotiffs/my-file.tif'
+    dest_dir = Path(tmp_path)
+
+    ut.S3_CLIENT.put_object(Bucket=s3_bucket, Key=download_path)
+
+    output_path = ut.download_file_from_s3(s3_bucket, download_path, dest_dir)
+
+    assert output_path == tmp_path / 'my-file.tif'
```

Comment:
`tmp_path` is already a `Path` object so no need to convert it: https://docs.pytest.org/en/stable/how-to/tmp_path.html

Also suggest renaming `download_path` -> `object_key` and adding an extra assertion.
```suggestion
    object_key = 'geotiffs/my-file.tif'

    ut.S3_CLIENT.put_object(Bucket=s3_bucket, Key=object_key)

    output_path = ut.download_file_from_s3(s3_bucket, object_key, tmp_path)

    assert output_path == tmp_path / 'my-file.tif'
    assert output_path.exists()
```